packages/base-driver/lib/express/server.js

```diff
@@ -242,31 +249,7 @@ function configureHttp({httpServer, reject, keepAliveTimeout}) {
     }
   );
 
-  appiumServer.on(
-    'connection',
-    /** @param {AppiumServerSocket} socket */ (socket) => {
-      socket.setTimeout(keepAliveTimeout);
-      socket.on('error', reject);
-
-      function destroy() {
-        socket.destroy();
-      }
-      socket._openReqCount = 0;
-      socket.once('close', () => serverState.notifier.removeListener('shutdown', destroy));
-      serverState.notifier.once('shutdown', destroy);
-    }
-  );
-
-  appiumServer.on('request', function (req, res) {
```

Comment:
I'm a little confused by this method; I assume it was unused?

Reply:
yes, to avoid them shadowing the top function arguments

Reply:
yes, calling process.exit would close any pending connections anyway, so we don't need all this dark magic anymore